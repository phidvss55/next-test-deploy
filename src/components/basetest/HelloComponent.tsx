import React from 'react'

interface Props {
  name?: string | undefined
}

const HelloComponent = ({ name }: Props) => {
  return (
    <div>
      <h1>Hello Component {name}</h1>
    </div>
  )
}

export default HelloComponent
