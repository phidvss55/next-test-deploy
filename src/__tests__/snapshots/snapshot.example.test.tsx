import HelloComponent from '@/components/basetest/HelloComponent'
import { render, screen } from '@testing-library/react'

describe('Testing snapshot example', () => {
  it('will check the matchers and pass', () => {
    const user = {
      createdAt: new Date(),
      id: Math.floor(Math.random() * 20),
      name: 'LeBron James'
    }

    expect(user).toMatchSnapshot({
      createdAt: expect.any(Date),
      id: expect.any(Number)
    })
  })
})
