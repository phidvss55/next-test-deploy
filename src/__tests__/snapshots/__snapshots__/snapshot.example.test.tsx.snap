// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing snapshot example will check the matchers and pass 1`] = `
{
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "LeBron James",
}
`;
